<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>771</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollArea {background color : white;}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1029</width>
      <height>769</height>
     </rect>
    </property>
    <widget class="QStackedWidget" name="FacilityINFO">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>730</width>
       <height>250</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_f1">
      <widget class="QLabel" name="Fac_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>610</width>
         <height>240</height>
        </rect>
       </property>
       <property name="text">
        <string>시설 정보</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Next_f1">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>20</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 14pt &quot;경기천년바탕 Regular&quot;;
    color: white;
    background-color: rgb(251, 86, 7);
    border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>다음</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_f2">
      <widget class="QLabel" name="Fac_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>610</width>
         <height>240</height>
        </rect>
       </property>
       <property name="text">
        <string>시설 정보</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Next_f2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>20</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 14pt &quot;경기천년바탕 Regular&quot;;
    color: white;
    background-color: rgb(251, 86, 7);
    border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>다음</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_f3">
      <widget class="QLabel" name="Fac_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>610</width>
         <height>240</height>
        </rect>
       </property>
       <property name="text">
        <string>시설 정보</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Next_f3">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>20</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 14pt &quot;경기천년바탕 Regular&quot;;
    color: white;
    background-color: rgb(251, 86, 7);
    border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>다음</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="bookPage">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>480</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="bookPage_1">
      <widget class="QLabel" name="bookpic1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>260</width>
         <height>300</height>
        </rect>
       </property>
       <property name="text">
        <string>추천도서</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="bookPage_2">
      <widget class="QLabel" name="bookpic2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>260</width>
         <height>300</height>
        </rect>
       </property>
       <property name="text">
        <string>추천도서</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="bookPage_3">
      <widget class="QLabel" name="bookpic3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>260</width>
         <height>300</height>
        </rect>
       </property>
       <property name="text">
        <string>추천도서</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="bookPage_4">
      <widget class="QLabel" name="bookpic4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>260</width>
         <height>300</height>
        </rect>
       </property>
       <property name="text">
        <string>추천도서</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="bookPage_5">
      <widget class="QLabel" name="bookpic5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>260</width>
         <height>300</height>
        </rect>
       </property>
       <property name="text">
        <string>추천도서</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="searchBook">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 14pt &quot;경기천년바탕 Regular&quot;;
    color: white;
    background-color: rgb(251, 86, 7);
    border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>도서정보 조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rentReturn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>340</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 14pt &quot;경기천년바탕 Regular&quot;;
    color: white;
    background-color: rgb(251, 86, 7);
    border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>대여 및 반납</string>
     </property>
    </widget>
    <widget class="QLabel" name="notice">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>421</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string>공지사항</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;경기천년바탕 Regular&quot;;</string>
     </property>
     <property name="text">
      <string>시설정보</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>420</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;경기천년바탕 Regular&quot;;</string>
     </property>
     <property name="text">
      <string>추천도서</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="Loginstack">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>10</y>
       <width>301</width>
       <height>71</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="btn_Login1234">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>191</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 14pt &quot;경기천년바탕 Regular&quot;;
    color: white;
    background-color: rgb(251, 86, 7);
    border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>로그인 / 회원가입</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="label_logined">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>경기천년바탕 Regular</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;경기천년바탕 Regular&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>491</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>일정</string>
      </attribute>
      <widget class="QLabel" name="even_cal">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>300</width>
         <height>330</height>
        </rect>
       </property>
       <property name="text">
        <string>이벤트</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>이벤트</string>
      </attribute>
      <widget class="QLabel" name="even_cal2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>480</width>
         <height>310</height>
        </rect>
       </property>
       <property name="text">
        <string>이벤트</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
